cmd/image-builder-maintenance/main.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/sirupsen/logrus"
+)
+
+func main() {
+	logrus.SetReportCaller(true)
+
+	conf := Config{
+		EnableDBMaintenance:    false,
+		DBEmailRetentionMonths: 24,
+	}
+
+	err := LoadConfigFromEnv(&conf)
+	if err != nil {
+		logrus.Fatal(err)
+	}
+
+	if conf.DryRun {
+		logrus.Info("Dry run, no state will be changed")
+	}
+
+	if !conf.EnableDBMaintenance {
+		logrus.Info("🦀🦀🦀 DB maintenance not enabled, skipping  🦀🦀🦀")
+		return
+	}
+	dbURL := fmt.Sprintf("postgres://%s:%s@%s:%s/%s?sslmode=%s",
+		conf.PGUser,
+		conf.PGPassword,
+		conf.PGHost,
+		conf.PGPort,
+		conf.PGDatabase,
+		conf.PGSSLMode,
+	)
+	err = DBCleanup(dbURL, conf.DryRun, conf.DBEmailRetentionMonths)
+	if err != nil {
+		logrus.Fatalf("Error during DBCleanup: %v", err)
+	}
+	logrus.Info("🦀🦀🦀 dbqueue cleanup done 🦀🦀🦀")
+}
```

Comment:
For code consistency and to be prepared, I'll stick to it until we have some good reasons to deviate from the composer implementation

Reply:
Wait, I think all of this should be logged into cloudwatch, and I think kibana logs anything in the namespace to kibana as well? not sure. But cloudwatch definitely.

Just not splunk since the hook isn't hooked up.

Reply:
I think so too, if not, I'd fix it in the deployment…